src/es/carbon-intensity.md

```diff
@@ -1,63 +1,61 @@
+> La intensidad de carbono puede ser 0 gCO<sub>2</sub>eq/kWh cuando la nueva carga puede ser proveída con oferta de una fuente renovable que de otro modo habría sido descartada.
 
-![alt_text](/assets/images/principles/carbon-intensity-3.png "There are moments when the marginal carbon intensity reaches 0")
+### Desplazamiento de la demanda
 
-> The marginal carbon intensity could be 0 gCO<sub>2</sub>eq/kWh when new load would be met with supply from a renewable source that would otherwise have been curtailed.
+En la actualidad no tenemos muchos métodos para almacenar o amortiguar los sistemas de redes eléctricas. Normalmente la electricidad se produce de modo que la oferta siempre cubra la demanda. Si se genera más energía de fuentes renovables que la necesaria para soportar la demanda, y todas nuestras opciones de almacenamiento están llenas, entonces descartamos (tiramos) esa energía limpia. Una solución es desplazar las cargas de trabajo a horas y ubicaciones donde haya mayor suministro de energías renovables, esto se llama *desplazamiento de la demanda*.
 
-### Demand Shifting
+Si puede ser flexible con el cuando y donde ejecuta sus cargas de trabajo, entonces puede elegir consumir electricidad cuando la intensidad de carbono es menor y parar cuando la intensidad de carbono es alta. Por ejemplo, entrenar un modelo de aprendizaje de máquinas a una hora distinta o en una región distinta donde la intensidad de carbono sea mucho menor.
 
-There is currently little in the way of storage or buffering in electrical grid systems. Normally electricity is produced so supply always meets demand. If more energy is being generated from renewables than is needed to support demand, and all our storage options are full, then we curtail (throw away) that clean energy. One solution is to shift workloads to times and locations where there is more supply of renewable energy, this is called *demand shifting*.
+> [Estudios](https://ieeexplore.ieee.org/document/6128960) han mostrado que estas acciones pueden resultar en una reducción de emisiones de carbono entre el 45% y 99% dependiendo del número de renovables en la red.
 
-If you can be flexible with when and where you run workloads then you can then choose to consume electricity when the carbon intensity is less and pause when carbon intensity is high. For example, training a machine learning model at a different time or region where the carbon intensity is much lower.
+Mire tu aplicación de extremo a extremo, identifique oportunidades para ser flexible respecto a las cargas de trabajo y use la intensidad de carbono de la electricidad como señal para cuando ejecutar esas cargas de trabajo.
```

Comment:
I suppose here is should be either "Mire su aplicación" or "Mira tu aplicación".
```suggestion
Mire su aplicación de extremo a extremo, identifique oportunidades para ser flexible respecto a las cargas de trabajo y use la intensidad de carbono de la electricidad como señal para cuando ejecutar esas cargas de trabajo.
```